src/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import HomePage from './pages/HomePage';
+import { Route, Routes } from 'react-router-dom';
+import AboutUs from './pages/AboutUs';
+import NotFound from './pages/NotFound';
+
+function App() {
+  return (
+    <>
+      <Routes>
+        <Route path={'/'} element={<HomePage />} />
```

Comment:
Here you can pass just string without wrapping {} in it. For example, <Route path='/' element={<HomePage />} /> like that

Reply:
Okey, thanks, i will change. To be honest, don't remember maybe ESlint warned me to wrap.